.circleci/config.yml

```diff
@@ -30,6 +30,11 @@ jobs:
           name: run tests
           command: yarn test
 
+      # run tests!
+      - run:
+          name: run tests
+          command: yarn test:e2e
+
```

Comment:
Hmm isn't this already in develop?

Reply:
Indeed, this was an issue from rebasing to bring in the develop branch.  Fixing this now.